superai/client.py

```diff
@@ -72,11 +72,12 @@ def request(
         method: str = "GET",
         query_params: dict = None,
         body_params: dict = None,
+        header_params: Optional[dict] = None,
```

Comment:
This is the class that is used by the sdk users, right? If so Could you add the new param to the end of the params list? If it stays like this, then it is a breaking change for the users who are not using keyword arguments.

Reply:
Good point, i put the new parameter last.